services/analytics/profile.go

```diff
@@ -117,6 +117,8 @@ func (s *Service) GetReport(ctx context.Context, in *qanpb.ReportRequest) (*qanp
 		return nil, fmt.Errorf("order column '%s' not in selected columns: [%s]", orderCol, strings.Join(uniqColumns, ", "))
 	}
 
+	search := strings.TrimSpace(in.Search)
```

Comment:
it's weird that we manipulate this variable in two places, it would be good to have both changes in the same place

Reply:
Moved